hasura/migrations/1606135484776_create_table_public_setting/up.sql

```diff
@@ -1 +1 @@
-CREATE TABLE "public"."setting"("id" serial NOT NULL, "eos_price" numeric NOT NULL, PRIMARY KEY ("id") );
+CREATE TABLE "public"."setting"("id" serial NOT NULL, "token_price" numeric NOT NULL, PRIMARY KEY ("id") );
```

Comment:
@JustinCast any migration should be edited because hasura track each migration to know witch one is already executed so this migration will works if you setup a new environment from the scratch but will ve ignored for other that already executed it with the original version like lacchain, jungle and mainnet

Reply:
this should be a new migration to alter the previous table and rename the column